website/content/en/docs/templates/github.md

```diff
@@ -0,0 +1,133 @@
+---
+title: GitHub template URLs
+weight: 20
+---
+
+| ⚡ Requirement | Lima >= 2.0 |
+|----------------|-------------|
```

Comment:
Also needs to be marked as experimental in this page and in https://lima-vm.io/docs/releases/experimental/